Makefile

```
@@ -1,4 +1,23 @@
-NUM_VALS = 3
+NUM_VALS=3
+
+docker-build:
+	@bash ./scripts/build_binary.sh
+
+init-testnet:
+	@bash ./scripts/init_chain.sh 
+
+start-docker-chain: docker-build init-testnet clean-docker-chain
```

Let's replace old ones. Also let's have a util functions inside `utils/node.py` to stop/start/pause nodes (instead of asking users to know more about docker, they can just call the python functions.
```suggestion
start-chain: docker-build init-testnet clean-docker-chain
```